src/App.tsx

```diff
@@ -1,14 +1,62 @@
 import './App.css';
 
-import AlbumCardList from './Components/AlbumCardList';
+import {useEffect, useState} from 'react';
+
+import AlbumList from './Components/AlbumList';
 import Header from './Components/Header';
-import {data} from './db';
+import SearchBar from './Components/SearchBar';
+import {Album} from './types/data';
 
```

Comment:
We could potentially create a baseURL variable. 

i.e

``` js 
const baseURL = "https://neuefische-spotify-proxy.vercel.app/api"
```
Which would allow us to do something like.

``` js
const URL =
    currentPage === 'FEATURE'
        ? '${baseURL}/featured'
        : `${baseURL}/search?artist=${queryName}`;
```